app/lunatech/lunchplanner/modules/Module.scala

```diff
@@ -0,0 +1,14 @@
+package lunatech.lunchplanner.modules
+
+/**
+  * Created by sandeep on 18/05/2017.
```

Comment:
Please remove this - should be visible in the git commits - so no need to add this info here.